lib/Doctrine/DBAL/ParameterType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Doctrine\DBAL;
+
+/**
+ * Contains statement parameter types.
+ */
+class ParameterType
+{
+    /**
+     * Represents the SQL NULL data type.
+     *
+     * @see \PDO::PARAM_NULL
+     */
+    public const NULL = 0;
```

Comment:
Uhhh... I can see more trouble here. We're using a lot of reserved names here.

Reply:
Discussed internally: if this ever breaks, there is a bac break in the language.